README.md

```diff
@@ -458,9 +458,9 @@ interface {
 }
 ```
 
-#### [Begin Firmware Update](https://zwave-js.github.io/node-zwave-js/#/api/node?id=beginfirmwareupdate)
+#### [Update Firmware](https://zwave-js.github.io/node-zwave-js/#/api/node?id=updatefirmware)
```

Comment:
should we still docuemnt the old command?

Reply:
I don't think so. It's deprecated and going to get removed in a future release. I left it for backwards compatibility but it should not be used by new clients